src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,6 +18,7 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
+        public static bool IsUap => IsWinRT || IsNetNative;
```

Comment:
The names are now confusing: we're mixing so many concepts... NetNative, WinRT, UWP, UAP... 

/cc @weshaggard @ericstj 
Why is NetNative referring to UWP only? I thought we're going to allow compilation of any kind of application (even non-Windows).